CONTRIBUTING.md

```diff
@@ -94,15 +96,17 @@ download and install anything:
 * On each documentation page, there should be an "Improve This Page" link at the very
   top.
 * Click on that link to open the respective source file (usually an `.rst` file in the
-  `doc` folder) on GitHub for editing online (you'll need a GitHub account).
+  `doc/` folder or a `.py` file in the `examples/` folder) on GitHub for editing online
+  (you'll need a GitHub account).
 * Make your desired changes.
 * When you're done, scroll to the bottom of the page.
 * Fill out the two fields under "Commit changes": the first is a short title describing
   your fixes; the second is a more detailed description of the changes. Try to be as
   detailed as possible and describe *why* you changed something.
-* Click on the "Commit changes" button to open a
-  [pull request (see below)](#pull-requests).
-* We'll review your changes and then merge them in if everything is OK.
+* Choose "Create a new branch for this commit and start a pull request" and
+  click on the "Propose changes" button to open a pull request.
```

Comment:
```suggestion
  click on the "Propose changes" button to open a pull request.
* The pull request will run the GMT automated tests and make a test deployment. You can see how your change looks   in the PyGMT documentation by clicking "View deployment" after the Vercel bot has finished (usually 5-10 minutes after the pull request was created).
```

It formatted in a strange way with my code suggestion, but I wanted to put someone in about how you can see how your change renders in the docs, per @meghanrjones suggestion.

Reply:
Done in 5a48be74d8e68c8213570b43761857c670e7d732.